pandas/tests/frame/test_repr_info.py

```diff
@@ -238,8 +238,8 @@ def test_info_duplicate_columns_shows_correct_dtypes(self):
         frame.info(buf=io)
         io.seek(0)
         lines = io.readlines()
-        assert 'a    1 non-null int64\n' == lines[3]
-        assert 'a    1 non-null float64\n' == lines[4]
+        assert '1. a    1 non-null int64\n' == lines[3]
+        assert '2. a    1 non-null float64\n' == lines[4]
```

Comment:
@bashtage : Do you still prefer 0-indexing?  I'm actually okay with 1-indexing because 0-indexing may not be as intuitive for non-purist programmers like ourselves :wink:

Reply:
I agree, I tilted with 1-indexing usage for `.info()` too.

Reply:
In terms of purist, I think it is more of a loss of information since 0 is meaningful while 1 is not.

Reply:
Thanks guys for working on this feature :). Is it ready in new version of pandas?